README.md

```diff
@@ -104,6 +105,13 @@ The AWS IoT Device Defender library has no dependencies on additional libraries
 
 See memory requirements for the latest release [here](https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/aws/device-defender-for-aws-iot-embedded-sdk/docs/doxygen/output/html/index.html#defender_memory_requirements).
 
+#### AWS IoT Over-The-Air Updates
```

Comment:
AWS IoT Over-the-air Updates

Reply:
Thanks, updated in https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1454/commits/1aa621d336055f201cddce7f917f67f38d609df2